conda-forge.yml

```diff
@@ -1,4 +1,6 @@
 build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
This feedstock isn't ready yet for cross-compilation, though I'd love to do that. I had tried in https://github.com/conda-forge/faiss-split-feedstock/pull/62

Reply:
Push a commit to the bot's PR to revert this and re-rendered below